scripts/hyperspectral/hyperspectral_workflow.sh

```diff
@@ -517,13 +518,13 @@ for ((fl_idx=0;fl_idx<${fl_nbr};fl_idx++)); do
 	typ_in_ENVI=$(grep '^data type' ${hdr_fl} | cut -d ' ' -f 4 | tr -d '\015')
 	xps_tm=$(grep 'current setting exposure' ${mtd_fl} | cut -d ':' -f 2 | tr -d '" ,\015' )
 	fl_clb="${drc_spt}/calibration_vnir_${xps_tm}ms.nc"
-	if [ "${wvl_nbr}" -eq 272 ]; then 
+	if [ "${wvl_nbr}" -eq 272 ] || [ "${wvl_nbr}" -eq 273 ]; then 
```

Comment:
in the broad sense that you define, all branch points in the HS workflow depend on metadata. there are three files with metadata for each raw image. if you count information programatically available elsewhere then the potential sources of metadata are even greater. the HS workflow attempts to be modular and use only local information from the smallest subset of files possible.